### PR TITLE
Update branding to 2.1.26

### PR DESCRIPTION
**Pull request by aik-jahoda**

### CODE REVIEW

File: dependencies.props [hunk 1]
```diff
@@ -5,14 +5,14 @@
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
       https://dotnetmygetlegacy.blob.core.windows.net/dotnet-core/index.json;
-      https://api.nuget.org/v3/index.json;
```
Comment (aik-jahoda) [inline dependencies.props:8]: @wtgodbe, is the feed url change expected?

File: tests/src/NuGet.Config [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <clear/>
     <add key="myget.org dotnet-core" value="https://dotnetmygetlegacy.blob.core.windows.net/dotnet-core/index.json" />
     <add key="nuget-build" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
```
Comment (aik-jahoda) [inline tests/src/NuGet.Config:10]: @wtgodbe, is the feed url change expected?

File: dependencies.props (hunk 1 above)
Comment (wtgodbe) [reply dependencies.props:8]: Yes

File: tests/src/NuGet.Config (hunk 2 above)
Comment (wtgodbe) [reply tests/src/NuGet.Config:10]: Yes

**Review by wtgodbe — APPROVED** [review]

**Review by Anipik — APPROVED** [review]

**Pull request merged**